articles/_includes/_embedded_login_warning.md

```diff
@@ -0,0 +1,3 @@
+::: warning
+Embedded login for web uses Cross Origin Authentication which [does not work reliably on all browsers](/cross-origin-authentication#limitations-of-cross-origin-authentication) if you do not enable [Custom Domains](/custom-domains). The use of Custom Domains is a paid feature. A good alternative is to [migrate to Universal Login](/guides/login/migration-embedded-universal) if you cannot use Custom Domains in your application.
```

Comment:
The first sentence does not make sense, is the `which` a typo?

Reply:
+ there is a conflict in the PR

Reply:
The conflict I can fix. The sentence sounds ok to me (it's a bit hard to read with the URLs thrown in, but rendered it looks ok). I'm unsure how I would change it. Depending on speech cadence, I guess we could add a comma before "which", but taking it out would make no sense to me.

Reply:
Fixed the conflict